packages/runner-ct/src/lib/webpack-hmr-client.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable no-console */
```

Comment:
Is this specific to webpack !!! If so, we wanted to avoid to bind ourself to webpack. 
Should we have this in a separate package?